### ISSUE TITLE
Insufficient colour contrast on the 'Changelog' webpage

### ISSUE THREAD

**Issue by MelanieTPX** [tpximpact/mhclg-oruk#88]
The red text on the current Changelog webpage doesn't have enough contrast against the foreground to meet WCAG 2.2 AA guidelines. It is 3.8:1 and should be 4.5:1 minimum for normal text.

https://openreferraluk-development.netlify.app/developers/changelog

Tested on [webaim ](https://webaim.org/resources/contrastchecker/)

![Image](https://github.com/user-attachments/assets/25d2c10c-a0f8-4795-9d70-52775bc7945f)

**Comment by gjhltn** [comment]
This is odd because Axe dent complain about it. Look - here is Axe not complaining...

![Image](https://github.com/user-attachments/assets/81d104ff-3d87-4b60-82b3-a0438c070050)

**Comment by MelanieTPX** [comment]
Updated colours now has contrast ratio of 9.15:1 which looks good! Thanks Guy!

![Image](https://github.com/user-attachments/assets/935e2410-2ed1-454d-9a81-0bf51f9151e1)
